service/controller/app/v1/resource/releasemigration/create.go

```diff
@@ -50,8 +64,14 @@ func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
 		r.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("release %#q helmV3 migration not started", key.ReleaseName(cr)))
 		r.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("installing %#q", migrationApp))
 
+		// cordon all charts except chart-operator
+		err := r.cordonChart(ctx, cc.Clients.K8s.G8sClient(), tillerNamespace)
```

Comment:
I don't get this part. I know the tiller namespace is different. But the chart CRs are always in the giantswarm namespace.

Reply:
You're right, 🤦 I mixed up, it only need to get secret & configmap.